packages/tokens/src/Tokens.stories.tsx

```diff
@@ -83,6 +85,78 @@ function SpacingBlock({ space }: { space: keyof typeof spacing }) {
   );
 }
 
+export const OverflowShadows = () => {
+  return (
+    <div
+      className={css`
+        display: flex;
+        flex-direction: column;
+        gap: ${spacing[400]}px;
+      `}
+    >
+      {Object.values(Theme).map((theme: Theme) => {
```

Comment:
remind me again, what did we agree on for the darkmode squares since it's hard to see the shadows?